docker/production/docker-compose.yaml

```diff
@@ -0,0 +1,92 @@
+services:
+
+  mysql:
+    image: mysql:8.0.35
+    container_name: mysql
+    restart: always
+    environment:
+      MYSQL_DATABASE: 'jtr'
+      MYSQL_USER: 'user'
+      MYSQL_PASSWORD: 'password'
```

Comment:
Da wäre Detlef Kremer aber sauer xD

Reply:
nee ohne scheiß: ist es möglich das Passwort (gerne auch später noch) so zu speichern, dass das 1. im Git nicht sichtbar ist und 2. das was anderes ist als password?

Reply:
also unser Datenschutzbeauftragter würde auch schon wieder im Dreieck springen^^
ja, ist mindestens deploymentseitig möglich (ohne k8s muss ich mal schauen, ob ich nen schöneren Weg finde, als Env Variablen händisch anzulegen, weil k8s is too much nur dafür), und das ist ja erstmal auch nur ein Template, das mir Leander freundlicherweise direkt dazugelegt hat, welches ich dann fürs Deployment verwenden kann.

weiß aber aktuell nicht, wie das im Code verwendet wird, also ob diese Werte da aktuell hardcoded sind, und dann auch im code Konfigurativ eingebaut werden müssten. Aber das kann ja noch geklärt werden.